python/plugins/db_manager/dlg_table_properties.py

```diff
@@ -322,3 +331,31 @@ def deleteIndex(self):
                 self.refresh()
             except BaseError as e:
                 DlgDbError.showError(e, self)
+
+    def createComment(self):
+        #Function that add a comment to the selected table
+        try:
+            #Using the db connector, executing de SQL query Comment on table
+            self.db.connector._execute(None, 'COMMENT ON TABLE "{0}"."{1}" IS E\'{2}\' '.format(self.table.schema().name, self.table.name ,self.viewComment.text()))
+        except DbError as e:
+            DlgDbError.showError(e, self)
+            return
+        self.refresh()
+        #Display successful message
+        QMessageBox.information(self, self.tr("Add comment"), self.tr("Table successfully commented"))
+
+
+    def deleteComment(self):
+        #Function that drop the comment to the selected table
+        try:
+            #Using the db connector, executing de SQL query Comment on table. Comment is void ''
+            self.db.connector._execute(None, 'COMMENT ON TABLE "{0}"."{1}" IS E\'\' '.format(self.table.schema().name, self.table.name))
```

Comment:
Shouldn't it be "is NULL" instead?

Reply:
Right, didn't saw that at first, i'll change it, ty.